swagger_meta.go

```diff
@@ -19,6 +19,14 @@
 //          type: apiKey
 //          name: X-Session-Token
 //          in: header
+//     sessionBearerToken:
+//          type: apiKey
+//          name: Authorization
+//          in: header
+//     sessionCookie:
+//          type: apiKey
```

Comment:
Thank you for the submission! Unfortunately, this notation does not work as it sends `Cookie: the-session-token` while the correct notation is `Cookie: key=value;` or rather `Cookie: ory_kratos_session=...;`. In my research, I could not find a possibility to set this using Swagger 2.0. Session cookies are only supported starting with Swagger 3.0 (https://swagger.io/docs/specification/authentication/cookie-authentication/) but go-swagger has no plants of moving to 3.0 unfortunately: https://github.com/go-swagger/go-swagger/issues/1122#issuecomment-575968499